src/Mvc/Mvc.Core/src/Microsoft.AspNetCore.Mvc.Core.csproj

```diff
@@ -48,6 +49,7 @@ Microsoft.AspNetCore.Mvc.RouteAttribute</Description>
   </ItemGroup>
 
   <ItemGroup Label="Sources packages">
+    <Reference Include="Microsoft.Extensions.HashCodeCombiner.Sources" />
```

Comment:
With correct `InternalsVisibleTo` attributes in ref/ assemblies, why are changes to this project necessary?